docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -47,12 +51,20 @@ To delete existing identity from filesystem realm:
 /subsystem=elytron/filesystem-realm=fsRealm:remove-identity(identity=alex)
 ----
 
+An operation is added if an integrity-enabled filesystem realm needs to update the key-store or the key-store alias. The following command will update the realm and all the identity signatures to utilize the new key-pair.
```

Comment:
Maybe this could be reworded to something like:

It's possible to update the `key-store` or `key-store-alias` to use for integrity checking using the `update-key-pair` command as shown below. This command will also update the signature in any existing identity files.